kitchensink/index.html

```diff
@@ -82,17 +82,17 @@
       <div role="navigation" class="app-header">
         <div class="container-fluid">
 
-          <div class="navbar-header">
+          <!-- <div class="navbar-header">
```

Comment:
Perhaps we just remove this instead of commenting.

Reply:
Made changes accordingly